src/ethereum_test_specs/tests/fixtures/chainid_cancun_state_test.json

```diff
@@ -0,0 +1,91 @@
+{
+    "000/my_chain_id_test/Cancun": {
+        "_info": {
+            "hash": "0x7ef2e4726bc3d6914311fa963437f5868ff87ae3ea047d260c6609a9e6c646bf",
+            "fixture_format": "state_test"
+        },
+        "env": {
+            "currentCoinbase": "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentGasLimit": "0x02540be400",
+            "currentNumber": "0x01",
+            "currentTimestamp": "0x03e8",
+            "currentRandom": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "currentDifficulty": "0x00",
+            "currentBaseFee": "0x07",
+            "currentExcessBlobGas": "0x00"
+        },
+        "blobSchedule": {
+            "cancun": {
```

Comment:
ok since we changin the state format anyway. I have rather radical proposal. 
if it is 1 test per file now. then it could have network field as blockchain test.
an no need to create subdict of cancun here and in other places.
and post can be postState directly without list of forks. 
and transaction could have no arrays.

Reply:
I like this idea, but whereas adding one new top-level field is a relatively easy change for client teams, larger modifications of the fixture format wouldn't be welcomed by them at this already somewhat hectic stage of development. 

Perhaps we can discuss and schedule this for a calmer period (hopefully in a few weeks)? How about creating a follow-up issue.

Reply:
Created the new issue, we can discuss further there.

Reply:
https://github.com/ethereum/execution-spec-tests/issues/1102